src/Mod/PartDesign/Gui/ViewProvider.h

```diff
@@ -78,6 +78,9 @@ class PartDesignGuiExport ViewProvider : public PartGui::ViewProviderPart, PartG
     virtual bool setEdit(int ModNum) override;
     virtual void unsetEdit(int ModNum) override;
 
+    static inline void manageEditMode(int &ModNum) {
+        if (ModNum == ViewProvider::Transform || ModNum == ViewProvider::Cutting) ModNum = ViewProvider::Default; }

```

Comment:
coding style, please add line breaks

Reply:
Nope. This is header file so I purposely chose a compact style.

Reply:
I missed the rule that header files use another style than CPPs. Where is it?

I don't have a strong opinion, it is marked in my text editor as too long, and linebreak is proposed. That's how I stumbled upon it.

Reply:
Don't know if this is a strong rule but in header file you expect to be able to quickly list all function prototypes, so using multiple lines would make it less easy to browse. ;)